stdlib/http/src/main/java/org/ballerinalang/net/http/WebSocketConstants.java

```diff
@@ -79,6 +79,13 @@
     public static final String CLIENT_RESPONSE_FIELD = "response";
     public static final String CLIENT_CONNECTOR_FIELD = "conn";
 
+    public static final String ORG_NAME_SEPARATOR = "/";
+    public static final String WEBSOCKET_PACKAGE_PATH = "ballerina" + ORG_NAME_SEPARATOR + "WebSocket";
+    public static final String WEBSOCKET_ERROR_DETAIL_RECORD_NAME = "Detail";
+    public static final String WEBSOCKET_ERROR_CODE = "{ballerina/websocket}WebSocketError";
```

Comment:
```suggestion
    public static final String WEBSOCKET_ERROR_CODE = "{" + FULL_PACKAGE_HTTP + "}WebSocketError";
```
Let's stick with `{ballerina/http}WebSocketError`